src/content/blog/rust-based-javascript-linters-are-feature-incomplete-right-now/index.mdx

```diff
@@ -0,0 +1,130 @@
+---
+description: "Explaining why the marginal speed gains from Rust linters aren't comparable to the full feature set of typescript-eslint."
+image:
+  alt: Josh smiling on stage and giving a presentation showing the "Type Checking React (v2)" from the linked slides.
+  src: react-miami-me-speaking.jpg
+pubDate: 2024-01-07
+tags: [
+	"biome",
+	"deno lint",
+	"eslint",
+	"linting",
+	"oxc",
+	"rslint"
+]
+title: "Rust-Based JavaScript Linters are Feature Incomplete Right Now"
+---
+
+One of 2023's biggest trends for web tooling was rewriting existing tooling in [Rust](https://www.rust-lang.org).
+Rust is a wonderful programming language that allows for shockingly fast binaries that still interop well with other web tools courtesy of WebAssembly.
+The speedsups seen in tools such as [swc](https://swc.rs) and [Turbopack](https://turbo.build/pack) are very exciting for fast development experiences.
+
+[Biome](https://biomejs.dev), [`deno lint`](https://lint.deno.land), [oxc](https://github.com/oxc-project/oxc), and [RSLint](https://rslint.org) are all projects that include at least a JavaScript/TypeScript linter written in Rust.
```

Comment:
[Todo] Similar to #140's reaching-outs, I'll reach out to the teams of each of these to make sure I'm not misrepresenting anything.

Reply:
I reached out privately. Shoutout https://github.com/JoshuaKGoldberg/github-username-to-emails & https://github.com/JoshuaKGoldberg/github-username-to-emails-site for saving time finding emails...

Since this is the last third of December, I'll ping folks again the first week of January. Just in case they're already out.